.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,47 @@
+name: 🐛 Bug Report
+description: Create a report to help us reproduce and fix the bug
+
+body:
+  - type: markdown
+    attributes:
+      value: >
+        #### Before submitting a bug, please make sure the issue hasn't been already addressed by searching through [the
+        existing and past issues](https://github.com/pytorch/torchdynamo/issues)
+  - type: textarea
+    attributes:
+      label: 🐛 Describe the bug
+      description: |
+        Please provide a clear and concise description of what the bug is.
+      placeholder: |
+        A clear and concise description of what the bug is.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Error logs
+      description: |
+        Please provide the error you're seeing
+      placeholder: |
+        Error...
+    validations:
+      required: true
+  
+  - type: textarea
+    attributes:
+      label: Minified repro
+      description: |
+        Please run the minifier on your example and paste the minified code below 
+        Learn more here https://github.com/pytorch/torchdynamo/blob/main/documentation/TROUBLESHOOTING.md
+      placeholder: |
+        >> torch._dynamo.config.repro_after="aot" python your_model.py
```

I think users need to do:

```suggestion
        env TORCHDYNAMO_REPRO_AFTER="aot" python your_model.py
```